### PR TITLE
add values for fsgroup-volume image

### PR DESCRIPTION
**Pull request by erbzn**
Signed-off-by: Kersten Schlosser <kersten.schlosser@mailbox.org>

### Description
Using persistence for opensearch with harbor as private repository will fail because busybox ist a single repository image. Therefore setting the image in values.yaml is needed.
 
### Issues Resolved
https://github.com/opensearch-project/helm-charts/issues/126
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by TheAlgo — APPROVED** [review]
LGTM!

**Review by TheAlgo — DISMISSED** [review]
LGTM! Except you need to increment the Chart version and update the Changelog. @erbzn Can you do that as well?

**Comment by erbzn** [comment]
Thank you @TheAlgo.
I've updated the Changelog and bumped Chart version to 1.3.1

File: charts/opensearch/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ---
 ## [Unreleased]
 ### Added
+- Added image definition for `fsgroup-volume` initContainer to `values.yaml`.
```
Comment (peterzhuamazon) [inline charts/opensearch/CHANGELOG.md:10]: Wrong edit, you should create a 1.3.1 block and add your change notes.
Thanks.

Comment (erbzn) [reply charts/opensearch/CHANGELOG.md:18]: My bad. Thank you @peterzhuamazon

Comment (peterzhuamazon) [reply charts/opensearch/CHANGELOG.md:18]: Thanks.

**Review by zelinh — APPROVED** [review]
Thanks!

**Review by peterzhuamazon — APPROVED** [review]
Thanks.

**Pull request merged**